Purchases/Public/RCPurchases.m

```diff
@@ -673,12 +675,30 @@ - (void)checkTrialOrIntroductoryPriceEligibility:(NSArray<NSString *> *)productI
                                  completionBlock:(RCReceiveIntroEligibilityBlock)receiveEligibility
 {
     [self receiptData:^(NSData * _Nonnull data) {
-        [self.backend getIntroEligibilityForAppUserID:self.appUserID
-                                          receiptData:data
-                                   productIdentifiers:productIdentifiers
-                                           completion:^(NSDictionary<NSString *,RCIntroEligibility *> * _Nonnull result) {
-                                               CALL_IF_SET_ON_MAIN_THREAD(receiveEligibility, result);
-                                           }];
+        LocalReceiptParser *receiptParser = [[LocalReceiptParser alloc] init];
+        [receiptParser checkTrialOrIntroductoryPriceEligibilityWithData:data
+                                                     productIdentifiers:productIdentifiers
+                                                             completion:^(NSDictionary<NSString *, NSNumber *> * _Nonnull receivedElegilibity,
```

Comment:
```suggestion
                                                             completion:^(NSDictionary<NSString *, NSNumber *> * _Nonnull receivedEligilibity,
```

Reply:
I've miss-typed eligibility so many times it's now even messed up in my muscle memory. nice catch.